### PR TITLE
Fixing CI config for Rust install in `dash`

### PR DESCRIPTION
**Pull request by HammadTheOne**
This PR adds some minor fixes to the CI config in relation to the addition of the `orjson` dependency in `dash-2.0.0`.

### CODE REVIEW

**Comment by alexcjohnson** [comment]
I still don't get this rust issue, feels like a red herring to me, and I don't want us to get in the habit of installing it everywhere "just in case". What are you seeing that makes this seem necessary? The `dash` and `plotly.py` tests run just fine, both install `orjson` in the same kind of environment, and both should be implicitly using it, just as we do here and in dashR.

cc @homer6

**Comment by HammadTheOne** [comment]
> I still don't get this rust issue, feels like a red herring to me, and I don't want us to get in the habit of installing it everywhere "just in case". What are you seeing that makes this seem necessary? The dash and plotly.py tests run just fine, both install orjson in the same kind of environment, and both should be implicitly using it, just as we do here and in dashR.

Seems like just upgrading pip is an easier fix.

File: .circleci/config.yml [hunk 1]
```diff
@@ -108,6 +111,7 @@ jobs:
                 name: Install Dash
                 command: |
                     . venv/bin/activate
+                    pip install --upgrade pip
```
Comment (alexcjohnson) [inline .circleci/config.yml:114]: Here you're in the same venv as [line 102](https://github.com/plotly/dash-bio/pull/570/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R102) so this should be redundant

**Review by alexcjohnson — APPROVED** [review]
💃

**Pull request merged**